tests/_data/Common/Bridge/BridgeMethodsTest/testRuleDoesNotApplyWhenBridgeMethodsAreCorrect.php

```diff
@@ -44,5 +44,5 @@ class DataObject
 
 interface CacheClientInterface
 {
-    public function save(string $key, DataObject $data, string $prefix = null);
-}
\ No newline at end of file
+    public function save(string $key, DataObject $data, $prefix = null, array $options = null);
```

Comment:
What is the parent interface signature here? Can you paste it please for easier review on those changes?

Reply:
We can also keep stricter param types for now, though.